runner/estimate_model_test.go

```diff
@@ -7,44 +7,53 @@ import (
 )
 
 func TestDefaultEstimateModel(tt *testing.T) {
-	t := wrapt.WrapT(tt)
 
-	r := RunSettings{
-		OutputDir: "{{ .Name }}",
-	}
+	testId := "UNIT-RUN-003"
+	tt.Run(testId, func(tt *testing.T) {
```

Comment:
I added these `tt.Run()` calls so that I could get the `testId` into the output, but I'm not 100% confident this is the right way to do it. Any input is welcome.

Reply:
It will give you the sub test name after spending a slash to the outer test function name. I think a dash works here. Spaces in a test name are concerned to underscores, btw.